pkg/server/api/agent/v1/service_test.go

```diff
@@ -40,6 +142,9 @@ func setupServiceTest(t *testing.T) *serviceTest { //nolint: unused,deadcode
 		logHook: logHook,
 	}
 
+	ctx := context.Background()
+	test.createTestNodes(ctx, t)
```

Comment:
not sure if we want to always create attested nodes, it will affect future tests